packages/block-library/src/site-title/editor.scss

```diff
@@ -2,7 +2,3 @@
 	padding: 1em 0;
 	border: 1px dashed;
 }
-
-.editor-styles-wrapper :where(.wp-block-site-title a) {
-	color: inherit;
-}
```

Comment:
Removed because there's a rule in the style.scss that does the same thing (and has the same specificity, so exactly the same).